packages/fxa-auth-server/test/client/api.js

```diff
@@ -944,7 +944,7 @@ module.exports = (config) => {
     });
   };
 
-  ClientApi.prototype.recoveryPhoneNumberCreate = async function (
+  ClientApi.prototype.recoveryPhoneCreate = async function (
```

Comment:
If this changes, probably all of them should change to be consistent.

Reply:
Good point, I'll revert this but will delete the duplicate entry in `client/index.js` (just noticed)